docs/oscar_references.bib

```diff
@@ -1067,6 +1067,47 @@ @Book{Ful98
   doi           = {10.1007/978-1-4612-1700-8}
 }
 
+@Article{FKRS20,
+	author				= {Fruehbis-Krueger, Anne and Ristau, Lukas and Schober, Bernd},
+	title					= {Embedded desingularization for arithmetic surfaces -- toward a parallel implementation}, 
+  year					= {2020},
+	pages					= {32},
+  eprint				= {1712.08131},
+  archivePrefix	= {arXiv},
+  primaryClass	= {math.AG},
+  url						= {https://arxiv.org/abs/1712.08131}, 
+}
+
+@Article{Hasse1937,
+	author 				= {Hasse, H.},
+	journal 			= {Journal für die reine und angewandte Mathematik},
+	keywords 			= {algebraic function fields; higher differential quotients},
+	pages 				= {215-223},
+	title 				= {Noch eine Begründung der Theorie der höheren Differentialquotienten in einem algebraischen Funktionenkörper einer Unbestimmten. (Nach einer brieflichen Mitteilung von F.K. Schmidt in Jena).},
+	url 					= {http://eudml.org/doc/150015},
+	volume 				= {177},
+	year 					= {1937},
+}
+
+@Book{Cut04,
+  title					= {Resolution of Singularities},
+  author				= {Cutkosky, S.D.},
+  isbn					= {9780821872383},
+  series				= {Graduate studies in mathematics},
+  url						= {https://books.google.de/books?id=OkAppJ7dXsgC},
+  publisher			= {American Mathematical Soc.}
+}
+
+@Misc{Haze11,
+	title					= {Hasse-Schmidt derivations and the Hopf algebra of noncommutative symmetric functions}, 
+	author				= {Michiel Hazewinkel},
+	year					= {2011},
+	eprint				= {1110.6108},
+	archivePrefix	= {arXiv},
+	primaryClass	= {math.RA},
+	url						= {https://arxiv.org/abs/1110.6108}, 
+}
+
```

Comment:
bibtool is unhappy about the formatting of this file. Please see https://docs.oscar-system.org/dev/DeveloperDocumentation/documentation/#Updating-the-bibliography for more information about how to format the bibliography